common/models/user.js

```diff
@@ -673,6 +673,49 @@ module.exports = function(User) {
       next();
     });
 
+    // Delete old sessions once email is updated
+    UserModel.observe('before save', function beforeEmailUpdate(ctx, next) {
+      if (ctx.isNewInstance) return next();
+      if (ctx.where) {
+        ctx.Model.find({ where: ctx.where }, function(err, userInstances) {
+          if (err) return next(err);
+          getOriginalUserData(userInstances);
+        });
+      } else if (ctx.instance) {
+        ctx.hookState.id = ctx.instance.id;
+        var oldId = ctx.hookState.id;
```

Comment:
AFAICT, `ctx.hookState.id` is not used anywhere else, please remove.

Also the name `oldId` is a bit misleading. The id is (almost) never changed, therefore there is no such thing as "old" and "new" id.

I am proposing to use `id` or `userId` instead.